service/src/main/java/de/adorsys/psd2/sandbox/xs2a/web/RedirectController.java

```diff
@@ -94,8 +94,8 @@ public String handleConsentCreationRedirectRequest(
       @RequestParam("psu-id") String psuId,
       Model model) {
 
-    Optional<OnlineBankingData> data = redirectService.getOnlineBankingDataForConsent(externalId);
     redirectService.handleConsentCreationRedirectRequest(externalId, psuId);
+    Optional<OnlineBankingData> data = redirectService.getOnlineBankingDataForConsent(externalId);
```

Comment:
Why did no test fail?

Reply:
Because we only displayed it wrong and there are no UI test for the redirect controller.